solutions/usaco-1011.mdx

```diff
@@ -95,4 +95,58 @@ public class triangles{
 
 </JavaSection>
 
+<CPPSection>
+
+```CPP
+
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
```

Comment:
Can you add spaces between the for/if's and opening parentheses

Reply:
done